### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by DavidHewWing**
Added logging to morgan.
- added logging to the error-handlers.

### CODE REVIEW

**Comment by DavidHewWing** [comment]
@Wuon yes I'm looking into it

File: src/middlewares/error-handler.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 
 const httpStatus = require('http-status');
+const logger = require('../utils/logger');
 
 // hanlde not found error
 exports.handleNotFound = (req, res, next) => {
+  logger.error(`Request resource not found | ${req.method} | ${req.originalUrl}`);
```
Comment (callmekungfu) [inline src/middlewares/error-handler.js]:
```suggestion
  logger.error(`Request resource not found | 404 | ${req.method} | ${req.originalUrl}`);
```

File: src/utils/logger.js [hunk 2]
```diff
@@ -28,6 +28,12 @@ const logger = createLogger({
 logger.add(new transports.File({ filename: errLog, level: 'error' }));
```
Comment (Wuon) [inline src/utils/logger.js]: winston.transports.Console() for dev only or below?

Comment (DavidHewWing) [reply src/utils/logger.js]: we are currently doing that

```
if (process.env.NODE_ENV !== 'production') {
  logger.add(new transports.Console({
    level: 'debug',
  }));
}
```

**Pull request merged**